presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -164,4 +187,31 @@ private String indentString(int indent)
     {
         return Strings.repeat("    ", indent);
     }
+
+    private static class AnyQualifiedNameReference extends QualifiedNameReference
```

Comment:
I've changed it to AnyExpression, since function arguments are expressions, not necessarily references.